tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/jvmmodel/TclJvmModelInferrer.xtend

```diff
@@ -38,7 +39,8 @@ class TclJvmModelInferrer extends AbstractModelInferrer {
 	@Inject extension JvmTypesBuilder
 	@Inject extension TclModelUtil
 	@Inject IQualifiedNameProvider nameProvider
-	
+	@Inject extension TclAssertCallBuilder
```

Comment:
I like the extraction in a separate class, but I would remove the `extension` and specify the name to `assertBuilder`